position/sync/chooseOneResults.js

```diff
@@ -0,0 +1,30 @@
+var isArray = require('isarray')
+var Position = require('../../position/sync/position')
+
+//Expects `poll` and `position` objects passed in to be of shape: 
+//{
+//  value: {
+//    content: {...},
+//    timestamp: ...
+//    author: ...
+//  }
+//}
+//
+//postions must be of the correct type ie: type checked by the caller.
+module.exports = function ({positions, poll}) {
+  return positions.reduce(function (results, position) {
+    var { positionDetails: {choice} } = Position(position)
+
+    if (choice >= poll.pollDetails.choices.length || position.value.timestamp > poll.closesAt) {
```

Comment:
I reckon making these 2 standalone functions would be nice for future: 
- `!validChoice(poll, choice)`
- `!positionBeforeClose(poll, position)` - this would be re-useable in multiple files

Reply:
done